packages/web-react/src/components/Tooltip/useFloating.ts

```diff
@@ -54,12 +58,27 @@ export const useFloating = (props: UseTooltipUIProps) => {
   } = props;
 
   const [maxWidth, setMaxWidth] = useState<number | undefined>(undefined);
+  const [isClicked, setIsClicked] = useState(false);
   const mainAxisOffset = cornerOffset + tooltipArrowWidth;
 
   // Floating UI library settings
   const { x, y, refs, context, placement, middlewareData } = useFloatingUI({
     open: isOpen,
-    onOpenChange: onToggle,
+    onOpenChange: (open, event, reason) => {
+      if (enableHover) {
+        // if tooltip is opened by click, do not close until clicked again or outside press or escape key
+        if (reason === 'click') setIsClicked((prev) => !prev);
```

Comment:
Please, use parentheses `{...}` for better readability.

Reply:
added